requestsEx.py

```diff
@@ -0,0 +1,25 @@
+import requests
+import sys
+
+
+def top_repository_from_github(programming_language: str, number: int):
```

Comment:
do you return a single repository or multiple repositories? change function name